src/main/webapp/assets/js/feedScript.js

```diff
@@ -123,6 +129,8 @@ async function onLoad() {
   modalForm = document.getElementById('modal-form');
   toggleLegendButton = document.getElementById('toggle-legend-button');
   modalCard = document.getElementById('modal-create-post');
+  modalSubmitted = document.getElementById('modal-submitted');
+  modalSubmittedText = document.getElementById('modal-submitted-title');
```

Comment:
Should the id for this be modal-submitted-text as well?

Reply:
There is no `modal-submitted-text` id, but I can change the name of the variable to `modalSubmittedTitle` to make it more intuitive!